okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -842,6 +842,19 @@ public String fragment() {
   }
 
   /**
+   * Returns the redacted URL that would only show the domain name.
+   */
+  public String redactedUrl() {
```

Comment:
For this particular use case we are trying to retrieve the redacted Url and therefore I decided to go with return a String and keeping the name `redactedUrl()`

But if we decide to return a HttpUrl instead, then `redact()` will be more appropriate. What do you think?